### PR TITLE
SEO-187032-Missing-H1-FF

### PR DESCRIPTION
**Pull request by EsterAtieno**
Hi @Yvone-Atieno ,
Have made changes,Kindly review.

Title | Description
-- | --
Task Category | Missing H1
Content Task Link/Mail Screenshot | NA
Hotfix | https://github.com/syncfusion-content/fileformat-docs/pull/2163
Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187032/
Work link | https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B2898E97F-22C5-48F0-B677-88AB8168596C%7D&file=Ester.xlsx&action=default&mobileredirect=true

Changes Made | Reason for change | Reason for not identifying earlier
-- | -- | --
Redirect link | To improve accessibility |  

Regards,
Ester.

### CODE REVIEW

File: File-Formats/PDF/Assemblies-Required.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ control: PDF
 documentation: UG
 keywords: Assemblies
 ---
-# Assemblies Required to work with PDF 
+# Assemblies Required to work with .NET PDF
```
Comment (Yvone-Atieno) [inline File-Formats/PDF/Assemblies-Required.md:9]: Use this - Assemblies Required to work with PDF in .NET

File: File-Formats/PDF/Overview.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ control: PDF
 documentation: UG
 ---
 
-# Overview of PDF Framework
+# Overview of .NET PDF Framework
```
Comment (Yvone-Atieno) [inline File-Formats/PDF/Overview.md:9]: Use - Overview of PDF Framework in .NET

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: File-Formats/PDF/Assemblies-Required.md (hunk 1 above)
Comment (EsterAtieno) [reply File-Formats/PDF/Assemblies-Required.md]: Changed

File: File-Formats/PDF/Overview.md (hunk 2 above)
Comment (EsterAtieno) [reply File-Formats/PDF/Overview.md]: Changes solved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request closed without merging**